redfish-core/lib/oem/ibm/lamp_test.hpp

```diff
@@ -28,6 +28,11 @@ inline void getLampTestState(const std::shared_ptr<bmcweb::AsyncResp>& aResp)
                 getObjectType) {
             if (ec)
             {
+                if (ec.value() == 5) //generic:5 error
+                {

```

Comment:
Please review your code first. This is missing a }

Reply:
"}" is  there below, I have some editor setting issues, it seems. I'll push it back once ready.

Reply:
Looks good, ignore my commit